rules/S6500/metadata.json

```
@@ -1,5 +1,5 @@
 {
-  "title": "Installing unnecessary packages is security-sensitive",
+  "title": "Installing recommended packages automatically is security-sensitive",
```

I would have moved the "automatically" at the beginning. But I am not really sure.

```suggestion
  "title": "Automatically installing recommended packages is security-sensitive",
```

---

Yea, that looks nicer.